tests/test_decorators.py

```diff
@@ -813,15 +813,13 @@ def make_item(foo, bar):
     with pytest.raises(ValidationError) as excinfo:
         schema.dump(make_item(6, 1))
     errors = excinfo.value.messages
-    assert "foo" in errors
```

Comment:
Since we're not supporting validation-on-dump, we might want to just remove the `(pre|post)_dump` methods in this test and the assertions associated them, full stop.